release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1930,6 +1935,45 @@ With this release, the CCO no longer checks for the nonexistent permission.
 
 * Previously, proxy information that was set in the `install-config.yaml` file was not applied to the bootstrap process. With this release, proxy information is applied to bootstrap ignition data, which is then applied to the bootstrap machine. (link:https://issues.redhat.com/browse/OCPBUGS-12890[*OCPBUGS-12890*])
 
+* Previously, the {ibm-power-server-name} platform had no Dynamic Host Configuration Protocol (DHCP) network name, the DHCP resource was not deleted. With this release, a check looks for any DHCP resources with an `ERROR` state and deletes them so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-35224[*OCPBUGS-35224*])
+
+* Previously, when creating a Container-Native Storage API (CAPI) cluster on installer-provisioned infrastructure for {ibm-power-server-name}, the load balancer would become busy and stall. With this release, you can use the 'AddIPToLoadBalancerPool' command in a `PollUntilContextCancel' loop to restart the load balancer. (link: https://issues.redhat.com/browse/OCPBUGS-35088[*OCPBUGS-35088*])
```

Comment:
Container-Native Storage API isn't CAPI, something got mixed up here. Looking at the edit history in the bug, I think it's this:
```suggestion
* Previously, when creating a {ibm-power-server-name} cluster on installer-provisioned infrastructure by using the Cluster API, the load balancer would become busy and stall. With this release, you can use the 'AddIPToLoadBalancerPool' command in a `PollUntilContextCancel' loop to restart the load balancer. (link:https://issues.redhat.com/browse/OCPBUGS-35088[*OCPBUGS-35088*])
```

Reply:
:heavy_check_mark:

Reply:
:heavy_check_mark: